src/backend/routers/iep.test.ts

```diff
@@ -23,24 +23,30 @@ test("basic flow - add/get goals, subgoals, tasks", async (t) => {
 
   await trpc.iep.addSubgoal.mutate({
     goal_id: goal1!.goal_id,
+    status: "In Progress",
     description: "subgoal 1",
+    setup: "Setup here",
     instructions: "instructions here",
-    target_max_attempts: 5,
     materials: "materials",
-    target_level: 100,
-    baseline_level: 20,
+    target_level: 85,
+    baseline_level: 60,
```

Comment:
May be good to add a test for current_level as well?

Reply:
That makes sense! Thanks. I think a comment (or no changes) should be okay.

Reply:
Perfect! I added a comment there and to the initial_migrations to clarify that. I kept the test I added this morning and made sure to check that the value of the returned current_level was null, to make sure it wasn't getting a value but was still being included in the returned object.